allure-generator/package.json

```diff
@@ -77,19 +82,26 @@
     "split.js": "^1.3.5",
     "underscore": "^1.8.3"
   },
-  "jest": {
-    "setupFiles": [
-      "<rootDir>/src/test/javascript/bootstrap.js"
-    ],
-    "modulePaths": [
-      "<rootDir>/src/main/javascript"
-    ],
-    "transform": {
-      "^.+\\.hbs$": "<rootDir>/src/test/javascript/stubs/handlebars.js",
-      "^.+\\.js$": "babel-jest"
-    },
-    "moduleNameMapper": {
-      "\\.s?css$": "<rootDir>/src/test/javascript/stubs/css.js"
-    }
+  "engines": {
+    "node": ">=10.10.0"
+  },
+  "browserslist": "> 0.25%, not dead",
+  "scripts": {
+    "build": "npm run webpack:prod",
+    "cleanup": "rimraf build/www",
+    "coverage": "jest --coverage --collectCoverageFrom=src/main/javascript/**/*.js",
+    "lint": "eslint --ignore-path=.gitignore ./src/main/javascript",
+    "lint-fix": "eslint --fix --ignore-path=.gitignore ./src/main/javascript",
+    "test": "npm run lint",
+    "prettier:format": "prettier --write 'src/main/javascript/**/*.{ts,tsx,css,scss}'",
+    "start": "npm run webpack:dev",
+    "webpack": "node --max_old_space_size=4096 node_modules/webpack/bin/webpack.js",
+    "webpack-dev-server": "node --max_old_space_size=4096 node_modules/webpack-dev-server/bin/webpack-dev-server.js",
+    "webpack:build": "npm run cleanup && npm run webpack:build:main",
+    "webpack:build:main": "npm run webpack -- --config webpack/webpack.dev.js --env.stats=normal",
+    "webpack:dev": "npm run webpack-dev-server -- --config webpack/webpack.dev.js --inline --host=0.0.0.0 --port=3000 --env.stats=minimal",
+    "webpack:dev-verbose": "npm run webpack-dev-server -- --config webpack/webpack.dev.js --inline --port=3000 --profile --progress --env.stats=normal",
+    "webpack:prod": "npm run cleanup && npm run webpack:prod:main",
+    "webpack:prod:main": "npm run webpack -- --config webpack/webpack.prod.js --profile"
```

Comment:
I guess, you do not need that many different webpack commands. How about leaving only the `build`, like it was before?